docs/ruleset_functions/get_number_of_floors.md

```diff
@@ -0,0 +1,35 @@
+# get_number_of_floors
+**Schema Version:** 0.0.28  
+
+**Description:** gets the number of floors in the building.  Parking Garages are not counted
+
+**Inputs:**  
+- **RMI**: The baseline ruleset model instance
+
+**Returns:**  
+- **result**: an integer indicating the number of floors
+ 
+**Function Call:**
+
+## Logic:
+- check to see if Building.number_of_floors_above_grade and Building.number_of_floors_below_grade exisits, if so, the sum of these two numbers will be returned. `if Building.number_of_floors_above_grade != null && Building.number_of_floors_below_grade != null:`
+	- set result equal to the number of floors above grade plus the number of floors below grade: `result = Building.number_of_floors_above_grade + null && Building.number_of_floors_below_grade`
```

Comment:
This method does not account for the fact that there may be garage floors. Also, what us returned by the function if neither number of floors nor floor names are included in the RMD? Also, what if lighting_space_types are not entered, and we are not able to tell if any zone are garages?

Reply:
the note regarding `Building.number_of_floors_above_grade` and `Building.number_of_floors_below_grade` specifically states that parking-only floors should be excluded.

I would argue that if neither is entered, it is not a valid RMD.

Regarding the question about lighting_space_type - @weilixu can correct me here, but I believe that for each function, his team has a list of data elements that are required for the function to evaluate correctly, otherwise there will be some sort of error thrown.  Again, if this information is not provided, I don't think that we can evaluate section 18.

Reply:
Discussed at 6/1 call that this will not be the required fields in order for an RDSs to be valid. Weili confirmed that if these values are not entered, the RDS that require them will return "undetermined".

Reply:
Now that we agreed that we need to count number of floors that are not solely taken by garage and unconditioned space, the logic in line 16 can only work if we update instructions for the corresponding fields in the schema. Is that the plan?